### PR TITLE
update apl-nuc.rst to align with the ACRN v0.1

### PR DESCRIPTION
**Pull request by ailun258**
update ACRN getting started guide to align with ACRN v0.1 release. so that community to use Clear Linux 23690 which integrated ACRN V0.1

Signed-off-by: ailun258 <ailin.yang@intel.com>

### CODE REVIEW

**Comment by NanlinXie** [comment]
Can one of the admins verify this patch?

**Comment by ailun258** [comment]
@dbkinder  please help review and get approved

File: doc/getting-started/apl-nuc.rst [hunk 1]
```diff
@@ -36,13 +36,13 @@ on your platform. You'll need a network connection for your platform to
 complete this setup.
 
 .. note::
-   ACRN requires Clear Linux version 22140 or newer. The instructions below
-   have been validated with version 22140 and need some adjustment to work
+   ACRN v.01 or latest requires Clear Linux version 23690 or newer. The instructions below
```
Comment (dbkinder) [inline doc/getting-started/apl-nuc.rst]: ACRN release version is ``v0.1`` not ``v.01``

I'd recommend changing this note to be:

    ACRN v0.1 (and the current master branch) requires Clear Linux
    version 23690 or newer.  If you use a newer version of Clear Linux,
    you'll need to adjust the instructions below to reference the version
    number of Clear Linux you are using.

File: doc/getting-started/apl-nuc.rst [hunk 2]
```diff
@@ -61,11 +61,20 @@ complete this setup.
 
       # swupd autoupdate --disable
 
+#. if an old Clear Linux version installed already on your Hardware. We recommend that you upgrade Clear Linux to 23690
```
Comment (dbkinder) [inline doc/getting-started/apl-nuc.rst]: Change to:

    #. If you have an older version of Clear Linux already installed
        on your hardware, use this command to upgrade Clear Linux
        to version 23690 (or newer):

File: doc/getting-started/apl-nuc.rst [hunk 3]
```diff
@@ -61,11 +61,20 @@ complete this setup.
 
       # swupd autoupdate --disable
 
+#. if an old Clear Linux version installed already on your Hardware. We recommend that you upgrade Clear Linux to 23690 
+   or newer to use ACRN v0.1 or latest, Use this command
+   to upgrade your Clear Linux version to 23690 or <latest version>:
+
+   .. code-block:: none
+
+      # swupd update -m 23690 or <latest version>
```
Comment (dbkinder) [inline doc/getting-started/apl-nuc.rst]: change to:

    # swupd update -m 23690       # or newer version

File: doc/getting-started/apl-nuc.rst [hunk 4]
```diff
@@ -84,6 +93,10 @@ complete this setup.
       | kernel-pk          | Run the Intel "PK" kernel(product kernel source)  |
       |                    | and enterprise-style kernel with backports        |
       +--------------------+---------------------------------------------------+
+      | desktop            | desktop application, Weston included              |
```
Comment (dbkinder) [inline doc/getting-started/apl-nuc.rst]: At this point folks may not know what ``weston`` is, so a brief description would be good:

    graphical desktop application, weston
    compositing window manager included

File: doc/getting-started/apl-nuc.rst [hunk 5]
```diff
@@ -84,6 +93,10 @@ complete this setup.
...
+      +--------------------+---------------------------------------------------+
+      | openssh-server     | include openssh                                   |
```
Comment (dbkinder) [inline doc/getting-started/apl-nuc.rst]: a better description would be:

    server-side support for secure connectivity and
    remote login using the SSH protocol

**Comment by dbkinder** [comment]
I went ahead and incorporated these changes into #722

**Pull request closed without merging**